sky/cli.py

```diff
@@ -2115,7 +2119,10 @@ def logs(
 
     3. If ``--sync-down`` is specified, the logs of the job will be downloaded
     from the cluster and saved to the local machine under
-    ``~/sky_logs``. Mulitple job_ids can be specified.
+    ``~/sky_logs``. Multiple job_ids can be specified.
+
+    4. If the job fails or fetching the logs fails, the command will exit with
+    a non-zero return code.
```

Comment:
There is an inconsistency in behavior here if a job fails:
1. `sky logs` will exit with a return code 100
2. `sky logs --status` will exit with a return code 1

Due to back compat reasons, we should probably stick to error code 1 if job fails, and use 98/99 for not finished/not found.

Reply:
Updated `sky logs --status` to return more descriptive error codes.